app/code/community/Payone/Core/Model/Service/TransactionStatus/Execute.php

```diff
@@ -198,4 +262,21 @@ public function getMaxExecutionTime()
     {
         return $this->maxExecutionTime;
     }
-}
\ No newline at end of file
+
+    /**
+     * Handles all transaction status that were failed this cron job run
+     * and sends a reporting email to the configured recipient.
+     */
+    private function handleFailed()
+    {
+        $this->getFactory()->helperEmail()->send(
```

Comment:
Should we first check if there are any failed IDs?

Reply:
Thank you for the hint! Ofc we should do this :)